data/json/items/gun/faults_gun.json

```diff
@@ -21,11 +21,11 @@
     "requirements": {  }
   },
   {
-    "id": "fault_gun_clogged",
+    "id": "fault_gun_dirt",
     "type": "fault",
-    "name": "Clogging",
-    "description": "Clogging is usually caused by firing many blackpowder loads without cleaning; it prevents firing the gun until cleaned.  Some guns are more resistant to this than others.",
-    "time": "5 m",
+    "name": "Fouling",
+    "description": "Fouling is caused by firing gunpowder loads repeatedly, which reduces reliability and can eventually cause damage to the gun.",
```

Comment:
I suggest rephrasing this and the blackpowder fouling description to let the player know about the meaning of the dirt symbol, and also make it clear that these faults doesn't mean serious business until the symbol becomes high.